vite.config.ts

```diff
@@ -0,0 +1,27 @@
+import react from "@vitejs/plugin-react";
+import {defineConfig} from "vite";
+
+
+// https://vite.dev/config/
+export default defineConfig({
+    appType: "mpa",
+    base: "./",
+    build: {
+        assetsDir: "",
+        rollupOptions: {
+            output: {
+                manualChunks: {
+                    "monaco-editor": ["monaco-editor"],
+                },
+            },
+        },
+        sourcemap: true,
+    },
+    optimizeDeps: {
+        exclude: ["clp-ffi-js/worker"],
```

Comment:
Or the ClpFfiJs-worker.wasm file is not found when serving from the debug server.

Reply:
Note that according to the docs, `optimizeDeps` only works with the debug server (rather than the build as well) in the latest version of Vite.